Doc/library/ast.rst

```diff
@@ -146,6 +146,95 @@ Node classes
     Snakes <https://greentreesnakes.readthedocs.io/en/latest/>`__ project and
     all its contributors.
 
+
+.. _ast-root-nodes:
+
+Root nodes
+^^^^^^^^^^
+
+.. class:: Module(body, type_ignores)
+
+   A Python module.
+
+   *body* is a :class:`list` of the module's :ref:`ast-statements`.
+   *type_ignores* is a :class:`list` of the module's type ignore comments,
+   see :func:`ast.parse` for more details.
```

Comment:
```suggestion
   *body* is a :class:`list` of the module's :ref:`ast-statements`.

   *type_ignores* is a :class:`list` of the module's type ignore comments;
   see :func:`ast.parse` for more details.
```

* Put them on separate lines for easily scanability
* Fix grammar error (comma splice)